sdk/cmake/CMakeMacros.cmake

```diff
@@ -722,6 +722,39 @@ function(get_defines OUTPUT_VAR)
     set(${OUTPUT_VAR} ${__tmp_var} PARENT_SCOPE)
 endfunction()
 
+function(add_translatable_rc main_rc)
+    set(_CURRENT_TRANSLATION_MODULE ${main_rc} PARENT_SCOPE)
+    file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lang)
+endfunction()
+
+
+function(add_translations)
+    cmake_parse_arguments(__TRANSLATE "" "TEMPLATE" "TRANSLATIONS" ${ARGN})
+    set(__TRANSLATE_TEMPLATE_NAME ${__TRANSLATE_TEMPLATE})
+    set(__TRANSLATE_TEMPLATE ${CMAKE_CURRENT_SOURCE_DIR}/lang/${__TRANSLATE_TEMPLATE}.rc)
+    foreach(target_lang ${__TRANSLATE_TRANSLATIONS})
```

Comment:
I wonder if it would make sense to account for `I18N_DEFS` here, so that unnecessary rc files aren't even generated.
That would go well together with auto-generating all the `#ifdef LANGUAGE_XX_XX` `#include "lang/xx-XX.rc"`